conf/common.cfg

```diff
@@ -0,0 +1,27 @@
+[db]
+# type of database
+backend_grid = django.db.backends.mysql
```

Comment:
Can we split the grid and cloud db settings into their own sections, please. This isn't a very human-friendly layout otherwise.

e.g. `[db cloud]`, `[db grid]`. Then the options can just be `backened`, `hostname`, etc.